libs/httpsignature/nitro_test.go

```diff
@@ -0,0 +1,50 @@
+package httpsignature
+
+import (
+	"crypto"
+	"encoding/hex"
+	"os"
+	"testing"
+	"time"
+)
+
+func now() time.Time {
+	return time.Date(2023, time.March, 28, 12, 0, 0, 0, time.UTC)
+}
+
+func TestVerifyNitroAttestation(t *testing.T) {
+	pcr3 := "e48b6ac6bab30e3717d28c2c88f2ba8b614e454590eb00b26170eef0d707b5b8e3a97662c20b2ced6192d3aaa2f5e24e"
```

Comment:
Where's this hardcoded value coming from? Just want to make sure I'm understanding it correctly.

Reply:
it's from the sample doc itself, encoded to hex